packages/authorization-process/src/model/domain/toEvent.ts

```diff
@@ -338,3 +338,24 @@ export function toCreateEventProducerKeychainEServiceRemoved(
     correlationId,
   };
 }
+
+export function toCreateEventClientAdminRemoved(
+  client: Client,
+  adminId: UserId,
+  version: number,
+  correlationId: CorrelationId
+): CreateEvent<AuthorizationEventV2> {
+  return {
+    streamId: client.id,
+    version,
+    event: {
+      type: "ClientAdminRemoved",
```

Comment:
Is there a UI way or a different way from this internal one that allows us to remove the client admin ID? If yes, we could be more specific with the event name and do something like `ClientAdminRemovedBy...` / `ClientAdminRemovedAfter...` as we did with template update events